ui/app/templates/workspace/projects/new.hbs

```diff
@@ -0,0 +1,51 @@
+<AppBreadcrumbs/>
+<PageHeader @iconName="plus-plain">
+  <div class="title">
+    <h1>New Project</h1>
+  </div>
+</PageHeader>
+
+<form class="pds-form">
+  <div part="fields">
+    <fieldset class="pds-formFieldSet">
+      <div class="pds-formField">
+        <label for="project-name" class="pds-fieldName">
+          Project Name
+        </label>
+        <Pds::HelpText>
+          <p>The Project Name needs to match your waypoint.hcl configuration</p>
+        </Pds::HelpText>
+        <Pds::Input
+          @type="text"
+          id="project-name"
+          placeholder="my-new-project"
+          required
+          value={{this.model.name}}
+          {{on "input" (pick "target.value" (set this "model.name"))}}
+        />
+      </div>
+    </fieldset>
+    <fieldset class="pds-formFieldSet">
+      <div class="pds-formField">
+        <label for="create-git" class="pds-fieldName">
+          <Pds::Icon @type="git-repository"/> Git Configuration
+        </label>
+        <Pds::HelpText>
+          <p>Connect a git repository. If you skip this, you can always configure it later in project settings.</p>
+        </Pds::HelpText>
+        <Pds::CheckboxField
+          @checked={{this.createGit}}
+          @id="create-git"
+          {{on "change" (set this "createGit" (not this.createGit))}}
+        >
+          Connect a Git repository to this Project
+        </Pds::CheckboxField>
+
+      </div>
+    </fieldset>
+  </div>
+  <Pds::Button @variant="primary" {{on "click" this.saveProject}}>Next</Pds::Button>
```

Comment:
Heh, I said this before too so sorry (again) - moving the handller to the `form` tag's submit event gets you some UX & a11y niceties.

Reply:
Sorry, yeah I should have thought about those after your review on the other PR

Reply:
No worries! I just didn't want you to think I was a broken record for repeating things, I think both of these were going on a the same time too so that makes it trickier.